lib/board.rb

```diff
@@ -1,5 +1,5 @@
 class Board
-  attr_reader :board_array
+  attr_reader :current_board
```

Comment:
I ❤️ that you don't mention `array` anymore.

However, maybe we can find something better. How about `grid`?

Reply:
Thanks for the feedback. Made this change, and amended tests which failed.

Reply:
@sohbaker and did you fix the tests?

Reply:
@laurentqro all tests fixed